metaflow/decorators.py

```diff
@@ -257,6 +281,34 @@ class MyDecorator(StepDecorator):
                   pass them around with every lifecycle call.
     """
 
+    cli_options = {}
```

Comment:
named it `cli_options` because `options` seemed a like a fairly common attributes that user defined step decorators could also have